Source/dvlnet/tcp_server.h

```diff
@@ -66,6 +68,8 @@ class tcp_server {
 	std::array<scc, MAX_PLRS> connections;
 	buffer_t game_init_info;
 
+	std::unique_ptr<PacketError> ioHandlerResult;

```

Comment:
Should this be an `optional` rather than `unique_ptr` (throughout)?

Reply:
Feel free to merge once this is concluded

Reply:
Done